platform/android/export/export.cpp

```diff
@@ -846,30 +912,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		String plugins_names = get_plugins_names(get_enabled_plugins(p_preset));
 
-		Vector<String> perms;
-
-		const char **aperms = android_perms;
-		while (*aperms) {
-
-			bool enabled = p_preset->get("permissions/" + String(*aperms).to_lower());
-			if (enabled)
-				perms.push_back("android.permission." + String(*aperms));
-			aperms++;
-		}
-
-		PoolStringArray user_perms = p_preset->get("permissions/custom_permissions");
-
-		for (int i = 0; i < user_perms.size(); i++) {
-			String user_perm = user_perms[i].strip_edges();
-			if (!user_perm.empty()) {
-				perms.push_back(user_perm);
-			}
-		}
-
-		if (p_give_internet) {
-			if (perms.find("android.permission.INTERNET") == -1)
-				perms.push_back("android.permission.INTERNET");
-		}
+		Vector<String> perms = _get_permissions(p_preset, p_give_internet);
```

Comment:
Please make a PR that precedes this PR that is focused on refactoring (extracting) this into a new function

Reply:
Not sure why this is still showing up even after i merged the permissions branch with master